includes/class-wc-query.php

```diff
@@ -640,35 +480,80 @@ public function order_by_rating_post_clauses( $args ) {
 	 * @return array
 	 */
 	public function get_meta_query( $meta_query = array() ) {
-		if ( ! is_array( $meta_query ) )
+		if ( ! is_array( $meta_query ) ) {
 			$meta_query = array();
+		}
 
 		$meta_query[] = $this->visibility_meta_query();
 		$meta_query[] = $this->stock_status_meta_query();
+		$meta_query[] = $this->price_filter_meta_query();
+		$meta_query[] = $this->rating_filter_meta_query();
+
+		return array_filter( apply_filters( 'woocommerce_product_query_meta_query', $meta_query, $this ) );
+	}
+
+	/**
+	 * Return a meta query for filtering by price.
+	 * @return array
+	 */
+	private function price_filter_meta_query() {
+		if ( isset( $_GET['max_price'] ) || isset( $_GET['min_price'] ) ) {
+			$min          = isset( $_GET['min_price'] ) ? floatval( $_GET['min_price'] ) : 0;
```

Comment:
Minor but since these are not aligned with anything else they don't need to be indented as much.